src/features/student/views/StudentSkillView/StudentSkillView.test.ts

```diff
@@ -1,29 +1,51 @@
+import { mockedSkillDetailed } from '@/__mocks__/fixtures/student/skills.fixtures'
+import { createSkillDetailedHandler } from '@/__mocks__/msw/handlers/student/skills.handlers'
+import { server } from '@/__mocks__/msw/server'
+
 import { PageTitleStub } from '@/common/components/PageTitle/PageTitle.stub'
 import { studentEducationSkillsRoute, studentHomeRoute } from '@/features/student/routes'
 import StudentSkillView from '@/features/student/views/StudentSkillView/StudentSkillView.vue'
-import { mount, type VueWrapper } from '@vue/test-utils'
-import { BddTest } from 'tests/utils'
+import { flushPromises, type VueWrapper } from '@vue/test-utils'
+import { createPinia, setActivePinia } from 'pinia'
+import { BddTest, mountComponent } from 'tests/utils'
 
 BddTest().given('a student skill view', () => {
   let wrapper: VueWrapper<InstanceType<typeof StudentSkillView>>
 
-  const stubs = { PageTitle: PageTitleStub }
+  const stubs = {
+    PageTitle: PageTitleStub,
+    StudentSkillViewContainer: {
+      name: 'StudentSkillViewContainer',
+      props: ['skillDetailed'],
+      template: '<div class="student-skill-view-container-stub" />'
+    },
+  }
 
   beforeEach(() => {
     vi.clearAllMocks()
-
-    wrapper = mount(StudentSkillView, { global: { stubs } })
+    setActivePinia(createPinia())
+    const handler = createSkillDetailedHandler(mockedSkillDetailed)
+    server.use(handler)
+    wrapper = mountComponent(StudentSkillView, {
+      props: {
+        skillId: 'skill-1-1'
+      },
+      global: {
+        stubs
+      },
```

Comment:
```suggestion
      props: { skillId: 'skill-1-1' },
      global: { stubs },
```

Tu peux faire comme ça, pas besoin d'aérer à l'excès ;)